### PR TITLE
Integrate supported CC sites endpoint

### PR DESCRIPTION
**Pull request by narin**
## Description
Update FE to use new supported_site endpoint

## Testing done
Local and unit

## Acceptance criteria
- [ ] FE uses new endpoint

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/api/index.js [hunk 1]
```diff
@@ -2,8 +2,6 @@ import moment from 'moment';
 import { apiRequest } from 'platform/utilities/api';
 import environment from 'platform/utilities/environment';
 
-// This wil go away once we stop mocking api calls
-const TEST_TIMEOUT = navigator.userAgent === 'node.js' ? 1 : null;
```
Comment (narin) [inline src/applications/vaos/api/index.js]: This wasn't being used anymore after updating `getSupportingSites()` so I removed it.

File: src/applications/vaos/api/sites-supporting-var.json [hunk 2]
```diff
@@ -1,619 +1,1004 @@
...
+    {
+      "id": "541",
+      "type": "object_type",
+      "attributes": {
+        "name": "Cleveland",
+        "timezone": "US/Eastern"
+      }
+    },
+    {
+      "id": "983",
```
Comment (jbalboni) [inline src/applications/vaos/api/sites-supporting-var.json]: I think I would cut this file down to just these two: 983 and 984. The rest isn't that useful as mock data, I don't think.

**Review by jbalboni — APPROVED** [review]
Looks good to me, though I think you need to add a mock for this service for the e2e tests.

File: src/applications/vaos/tests/e2e/vaos-helpers.js [hunk 3]
```diff
@@ -284,6 +284,23 @@ function initAppointmentListMock(token) {
...
+          id: '983',
+          type: 'object_type',
+          attributes: {
+            name: 'Cheyenne',
+            timezone: 'US/Mountain',
+          },
+        },
+      ],
+    },
+  });
```
Comment (narin) [inline src/applications/vaos/tests/e2e/vaos-helpers.js]: @jbalboni I haven't worked much with e2e tests yet.  Is this what you had in mind, or do I need to do more?

Comment (jbalboni) [reply src/applications/vaos/tests/e2e/vaos-helpers.js]: Yep, this is what I mean.

**Review by rianfowler — APPROVED** [review]

**Pull request merged**